pkg/operator/api/api.go

```diff
@@ -57,19 +57,17 @@ var clusterAccessRules = []v1beta1.PolicyRule{
 
 type Cluster struct {
 	context   *clusterd.Context
-	placement k8sutil.Placement
-	Name      string
 	Namespace string
+	placement k8sutil.Placement
 	Version   string
 	Replicas  int32
 }
 
-func New(context *clusterd.Context, name, namespace, version string, placement k8sutil.Placement) *Cluster {
+func New(context *clusterd.Context, namespace, version string, placement k8sutil.Placement) *Cluster {
 	return &Cluster{
 		context:   context,
-		placement: placement,
-		Name:      name,
```

Comment:
I went with `namespace` because it makes it very clear that we only support one rook cluster per kubernetes namespace and it corresponds to the kubernetes namespace.